README.md

```diff
@@ -60,36 +62,94 @@ const ServiceEntityPage = ({ entity }: { entity: Entity }) => (
 )
 ```
 
-## How to use datadog plugin in Backstage
+## Obtain the dashboard URL and graph tokens from Datadog that you will need for your metadata. 
+
+* Login to your Datadog account.
+
+### To get the dashboard URL.
+
+* Navigate to the dashboards list by hovering over dashboards on the page's left-hand side and selecting the dashboard list.
+
+* Select a dashboard from this list.
+
+* Within the dashboard you have chosen, click the settings cog on the screen's right-hand side, circled in red.
+
+![dashboard](./docs/dd-dashboard.png?raw=true)
+
+
+* Copy the URL from the Sharing textbox.
+
+* This URL is the value you need for the `datadoghq.com/dashboard-url` annotation.
+
+![dashboard share](./docs/dd-dashboard-share.png?raw=true)
 
-The datadog plugin is a part of the Backstage sample app. To start using it for your component, you have to:
 
-1. Add an annotation to the YAML config file of a component.
-   - To display a dashboard/screenboard you have to paste its whole embed url:
-     ```yml
-     datadog/graph-url: <graph-url>
-     ```
-   - To display an embeddable graph as a widget you have to  paste its embed token from the share url:
-     ```yml
-     argo-cd/graph-token: <graph-token>
-     ```
-   - You can set the embeddable graph to any of the sizes provided by the datadog (defaults to 'medium' if not set):
-     ```yml
-     argo-cd/graph-size: 'small' | 'medium' | 'large' | 'x-large'
-     ```
+### To get the graph token.
 
-## Develop plugin locally
+* Click on the graph pencil, circled in red, from your dashboard.
 
-You can clone the plugin repo into the `plugins/` directory:
+![dashboard](./docs/dd-dashboard-2.png?raw=true)
 
-```sh
-git clone https://github.com/RoadieHQ/backstage-plugin-datadog.git datadog
+* Click on the Share tab, choose a timeframe, graph size and legend. Click generate the embedded code. 
+
+* Copy the token value that is circled in red.

```

Comment:
Maybe clarify this a bit. The whole iframe is circled but only the token value from within there is needed right?

Reply:
Even though you approved i made your suggested change as it improves the clarity to the reader